docs/containers/overview.md

```diff
@@ -25,7 +25,7 @@ To install the extension, open the Extensions view (`kb(workbench.view.extension
 
 ## Editing Docker files
 
-You can get IntelliSense when editing your `Dockerfile` and `docker-compose.yml` files, with completions and syntax help for common commands.
+You can get [IntelliSense](../editor/intellisense.md) when editing your `Dockerfile` and `docker-compose.yml` files, with completions and syntax help for common commands.
```

Comment:
Use /docs/editor/intellisense.md.
See https://github.com/microsoft/vscode-docs/commit/fe3ea28a9522ff20de393d15c6f120080bcd4f45

Reply:
I'll fix this after it's merged.